projects/fsastorefrontlib/src/core/my-account/facade/quote.service.ts

```diff
@@ -81,6 +105,30 @@ export class QuoteService {
       .unsubscribe();
   }
 
+  retrieveQuoteCheckout(quote: any) {
+    this.retrieveQuote(quote);
+    this.subscription.add(
```

Comment:
I think it's not a good practise to subscribe inside services, so maybe it is a bad idea to move redirection logic here. I know that in that case you have duplicated code but maybe we could discuss further how this would reflect on whole app.

Reply:
We should definitely think of a way to avoid subscribing in services, unless we unsubscribe immediately like in `loadQuoteDetails` method. But even then, we should discuss is it possible to somehow move that logic elsewhere.